src/executor/source/utils/logging/morganMiddleware.ts

```diff
@@ -0,0 +1,19 @@
+import morgan, { StreamOptions } from "morgan";
+import Logger from "./Logger";
+
+const stream: StreamOptions = {
+  write: (message) => Logger.http(message),
+};
+
+//Only log if development
+const skip = () => {
+  const env = process.env.NODE_ENV || "development";
+  return env !== "development";
```

Comment:
So the incoming HTTP requests are only logged in development, correct?

Reply:
Yes exactly. That beeing said it currently defaults to development, so in a production environment the NODE_ENV variable needs to be set to "production" or something like that